site/content/docs/custom-theme.md

```diff
@@ -0,0 +1,110 @@
+---
+title: How to modify your theme
+description: Learn how to add custom fonts and colors to your site
+type: Blog
+created: 2022-12-07
+---
+
+## Adding custom fonts to your site
+
+_Adding a google font_
+
+Let's say you want to add a font from fonts.google.com then you would perform the following steps: **I going to add a font, _Kanit_, to my website.**
+
+1. Go to fonts.google.com
+2. Select the font you want
+
+![[kanit.png]]
+
+3. Select the styles you want
+
+![[style.png]]
+
+4. Press `View selected families` at the top right
+5. Copy everything inside the `<style>` tags
+
+![[copystyle.png]]
+
+6. Paste what you copied at the top of the `global.css` file, which can be found in your flowershow directory's style folder - `.flowershow/site/styles/global.css`
+
+```css=
+@import url('https://fonts.googleapis.com/css2?family=Kanit:wght@900&display=swap');
+```
+
+7. Open the file `tailwind.config.js` in `.flowershow` and add the new font by extending the fontFamily property of `theme` as shown below
+
+```javascript
+module.exports = {
+  theme: {
+    extend: {
+      fontFamily: {
+        kanit: ["Kanit"],
+      },
+    },
+  },
+};
+```
+
+Once added, you would then be able to use them in your tailwind classes by prefexing with `font-[name of the font]` eg. `font-kanit` in the above case.
```

Comment:
I actually think this use-case is quite rare (i.e. i want to have a different font for just one area) and also covers an additional point which is using html and tailwind css in markdown files (a good topic for another short how to).

I think focus should be on changing font throughout site e.g. changing the serif or sans font which changes font for all headings.